File: app/src/main/java/com/ravisharma/playbackmusic/Activities/AboutActivity.java
package com.ravisharma.playbackmusic.Activities;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.TextView;

import com.ravisharma.playbackmusic.BuildConfig;
import com.ravisharma.playbackmusic.R;

public class AboutActivity extends AppCompatActivity {

    TextView appVersion;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_about);

        appVersion = findViewById(R.id.appVersion);
        String appVer = BuildConfig.VERSION_NAME;

        appVersion.setText("v".concat(appVer));
    }

    public void finishPage(View view) {
        finish();
    }
}

File: app/src/main/java/com/ravisharma/playbackmusic/activities/AboutActivity.kt
package com.ravisharma.playbackmusic.activities

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity
import com.ravisharma.playbackmusic.BuildConfig
import com.ravisharma.playbackmusic.databinding.ActivityAboutBinding

class AboutActivity : AppCompatActivity() {

    lateinit var binding: ActivityAboutBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityAboutBinding.inflate(LayoutInflater.from(this))
        setContentView(binding.root)

        val appVer = BuildConfig.VERSION_NAME
        binding.appVersion.text = "Ver: $appVer"
    }

    fun finishPage(view: View?) {
        finish()
    }
}